do.sh

```diff
@@ -68,14 +68,8 @@ install_deps() {
 }
 
```

Comment:
@ajbalogh should we retire OPENAPI_GEN altogether ? The JSON we were generating was readable and helped validate the YAML file using openapi generator.

Now it is only used to generate python SDK.

I've opened https://github.com/open-traffic-generator/models/issues/80 to generate readable JSON.